src/main/scala/org/virtuslab/beholder/filters/json/JsonFilterFieldsComponent.scala

```diff
@@ -169,23 +184,22 @@ trait JsonFilterFieldsComponent extends FilterFieldComponent with SeqParametersH
         override protected def filterOnColumn(column: Rep[T#Value])(dataSeq: Seq[T#Value]): Rep[Option[Boolean]] = {
           SeqParametersHelper.isColumnValueInsideSeq(column)(dataSeq)((column, data) => column.? === data)
         }
+        def filterSchema: Schema[Seq[T#Value]] = Schema.schemaForIterable
       }
     }
 
-    private implicit def rangeFormat[T: Format]: Format[FilterRange[T]] =
```

Comment:
Is this implicit defined somewhere else in scope?

Reply:
yes, moved it to `FilterRange` Companion object